src/main/java/anticope/rejects/modules/InteractionMenu.java

```diff
@@ -47,6 +46,12 @@ public class InteractionMenu extends Module {
             .action(this::onKey)
             .build()
     );
+    public final Setting<Boolean> useCrosshairTarget = sgGeneral.add(new BoolSetting.Builder()
+            .name("use-crosshair-target")
```

Comment:
there's no need to put that behind a setting, just completely replace the DebugRenderer

Reply:
this option is for who dont want to select target behind walls imo

Reply:
ah allright